torch/csrc/jit/tracer.cpp

```diff
@@ -154,6 +154,42 @@ PreTraceInfo preRecordTrace(std::string op, // TODO: make this a Symbol
   return info;  // RVO
 }
 
+PreTraceInfo preRecordPythonTrace(THPObjectPtr pyobj,
```

Comment:
I guess we should mention that this is copy pasted from above so people remember to update both.

Reply:
I refactored out the common code in preRecordTrace and preRecordPythonTrace